[CI] - Native builds GraalVM 19.3.1 - Java 8

This issue reports the status of the daily cron job that runs all the native tests with multiple GraalVM versions.

Don't close this issue manually. This issue will be closed when the build gets fixed. It will be re-opened if it fails again later. You can subscribe to be notify of the status.

    Build page: https://github.com/quarkusio/quarkus/actions

The build seems to be failing with a weird GraalVM error:

```
2020-04-06T06:32:51.6174808Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on GraalVM Version 19.3.1 CE
2020-04-06T06:32:52.2941535Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] docker run -v /home/runner/work/quarkus/quarkus/integration-tests/quartz/target/quarkus-integration-test-quartz-999-SNAPSHOT-native-image-source-jar:/project:z --env LANG=C --user 1001:115 --rm quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java8 -J-DCoordinatorEnvironmentBean.transactionStatusManagerEnable=false -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=1 -J-Duser.language=en -J-Dfile.encoding=UTF-8 --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime -H:+JNI -jar quarkus-integration-test-quartz-999-SNAPSHOT-runner.jar -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:-AddAllCharsets -H:-IncludeAllTimeZones -H:EnableURLProtocols=http,https --enable-all-security-services --no-server -H:-UseServiceLoaderFeature -H:+StackTrace quarkus-integration-test-quartz-999-SNAPSHOT-runner
2020-04-06T06:33:04.4034128Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]    classlist:  10,643.17 ms
2020-04-06T06:33:05.5659436Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]        (cap):   1,160.78 ms
2020-04-06T06:33:06.9632683Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]        setup:   3,038.65 ms
2020-04-06T06:33:10.4900171Z 06:33:10,475 INFO  [org.jbo.threads] JBoss Threads version 3.0.1.Final
2020-04-06T06:34:23.6027586Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]   (typeflow):  51,158.40 ms
2020-04-06T06:34:23.6032500Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]    (objects):  22,817.93 ms
2020-04-06T06:34:23.6033340Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]   (features):     931.22 ms
2020-04-06T06:34:23.6033811Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]     analysis:  76,345.65 ms
2020-04-06T06:34:26.3970202Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]     (clinit):   1,001.12 ms
2020-04-06T06:34:26.4255181Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]     universe:   2,821.10 ms
2020-04-06T06:34:36.4792188Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]      (parse):   9,928.43 ms
2020-04-06T06:34:53.5021524Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]     (inline):  16,649.25 ms
2020-04-06T06:35:55.7146350Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]    (compile):  62,207.91 ms
2020-04-06T06:35:57.9514477Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]      compile:  91,417.65 ms
2020-04-06T06:36:01.6448367Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]        image:   3,687.28 ms
2020-04-06T06:36:02.4479194Z [quarkus-integration-test-quartz-999-SNAPSHOT-runner:21]        write:     804.81 ms
2020-04-06T06:36:02.4511959Z Fatal error: java.lang.RuntimeException: java.lang.RuntimeException: host C compiler or linker does not seem to work: java.lang.RuntimeException: returned 1
2020-04-06T06:36:02.4513620Z 
2020-04-06T06:36:02.4519807Z Running command: cc -v -o /project/quarkus-integration-test-quartz-999-SNAPSHOT-runner -z noexecstack -Wl,--gc-sections -Wl,--dynamic-list -Wl,/tmp/SVM-8540717505311960636/exported_symbols.list -Wl,-x -L/tmp/SVM-8540717505311960636 -L/opt/graalvm/jre/lib -L/opt/graalvm/jre/lib/svm/clibraries/linux-amd64 /tmp/SVM-8540717505311960636/quarkus-integration-test-quartz-999-SNAPSHOT-runner.o /opt/graalvm/jre/lib/libzip.a /opt/graalvm/jre/lib/libnet.a /opt/graalvm/jre/lib/libjava.a /opt/graalvm/jre/lib/libnio.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libffi.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/liblibchelper.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libjvm.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libstrictmath.a -lstdc++ -lpthread -ldl -lz -lm -lrt
2020-04-06T06:36:02.4520241Z 
2020-04-06T06:36:02.4520694Z Using built-in specs.
2020-04-06T06:36:02.4527557Z COLLECT_GCC=cc
2020-04-06T06:36:02.4528007Z COLLECT_LTO_WRAPPER=/usr/libexec/gcc/x86_64-redhat-linux/8/lto-wrapper
2020-04-06T06:36:02.4528805Z OFFLOAD_TARGET_NAMES=nvptx-none
2020-04-06T06:36:02.4529225Z OFFLOAD_TARGET_DEFAULT=1
2020-04-06T06:36:02.4529519Z Target: x86_64-redhat-linux
2020-04-06T06:36:02.4530523Z Configured with: ../configure --enable-bootstrap --enable-languages=c,c++,fortran,lto --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=http://bugzilla.redhat.com/bugzilla --enable-shared --enable-threads=posix --enable-checking=release --enable-multilib --with-system-zlib --enable-__cxa_atexit --disable-libunwind-exceptions --enable-gnu-unique-object --enable-linker-build-id --with-gcc-major-version-only --with-linker-hash-style=gnu --enable-plugin --enable-initfini-array --with-isl --disable-libmpx --enable-offload-targets=nvptx-none --without-cuda-driver --enable-gnu-indirect-function --enable-cet --with-tune=generic --with-arch_32=x86-64 --build=x86_64-redhat-linux
2020-04-06T06:36:02.4530883Z Thread model: posix
2020-04-06T06:36:02.4531214Z gcc version 8.3.1 20190507 (Red Hat 8.3.1-4) (GCC) 
2020-04-06T06:36:02.4532157Z COMPILER_PATH=/usr/libexec/gcc/x86_64-redhat-linux/8/:/usr/libexec/gcc/x86_64-redhat-linux/8/:/usr/libexec/gcc/x86_64-redhat-linux/:/usr/lib/gcc/x86_64-redhat-linux/8/:/usr/lib/gcc/x86_64-redhat-linux/
2020-04-06T06:36:02.4533206Z LIBRARY_PATH=/usr/lib/gcc/x86_64-redhat-linux/8/:/usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/:/lib/../lib64/:/usr/lib/../lib64/:/usr/lib/gcc/x86_64-redhat-linux/8/../../../:/lib/:/usr/lib/
2020-04-06T06:36:02.4534087Z COLLECT_GCC_OPTIONS='-v' '-o' '/project/quarkus-integration-test-quartz-999-SNAPSHOT-runner' '-z' 'noexecstack' '-L/tmp/SVM-8540717505311960636' '-L/opt/graalvm/jre/lib' '-L/opt/graalvm/jre/lib/svm/clibraries/linux-amd64' '-mtune=generic' '-march=x86-64'
2020-04-06T06:36:02.4537378Z  /usr/libexec/gcc/x86_64-redhat-linux/8/collect2 -plugin /usr/libexec/gcc/x86_64-redhat-linux/8/liblto_plugin.so -plugin-opt=/usr/libexec/gcc/x86_64-redhat-linux/8/lto-wrapper -plugin-opt=-fresolution=/tmp/ccYCCUrQ.res -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s -plugin-opt=-pass-through=-lc -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s --build-id --no-add-needed --eh-frame-hdr --hash-style=gnu -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 -o /project/quarkus-integration-test-quartz-999-SNAPSHOT-runner -z noexecstack /usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/crt1.o /usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/crti.o /usr/lib/gcc/x86_64-redhat-linux/8/crtbegin.o -L/tmp/SVM-8540717505311960636 -L/opt/graalvm/jre/lib -L/opt/graalvm/jre/lib/svm/clibraries/linux-amd64 -L/usr/lib/gcc/x86_64-redhat-linux/8 -L/usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64 -L/lib/../lib64 -L/usr/lib/../lib64 -L/usr/lib/gcc/x86_64-redhat-linux/8/../../.. --gc-sections --dynamic-list /tmp/SVM-8540717505311960636/exported_symbols.list -x /tmp/SVM-8540717505311960636/quarkus-integration-test-quartz-999-SNAPSHOT-runner.o /opt/graalvm/jre/lib/libzip.a /opt/graalvm/jre/lib/libnet.a /opt/graalvm/jre/lib/libjava.a /opt/graalvm/jre/lib/libnio.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libffi.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/liblibchelper.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libjvm.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libstrictmath.a -lstdc++ -lpthread -ldl -lz -lm -lrt -lgcc --as-needed -lgcc_s --no-as-needed -lc -lgcc --as-needed -lgcc_s --no-as-needed /usr/lib/gcc/x86_64-redhat-linux/8/crtend.o /usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/crtn.o
2020-04-06T06:36:02.4542225Z collect2: error: ld returned 1 exit status
2020-04-06T06:36:02.4542679Z 
2020-04-06T06:36:02.4543298Z 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
2020-04-06T06:36:02.4543578Z 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
2020-04-06T06:36:02.4544193Z 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
2020-04-06T06:36:02.4544482Z 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
2020-04-06T06:36:02.4544709Z 	at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:593)
2020-04-06T06:36:02.4545280Z 	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1005)
2020-04-06T06:36:02.4545658Z 	at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:462)
2020-04-06T06:36:02.4546048Z 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:315)
2020-04-06T06:36:02.4546601Z 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:454)
2020-04-06T06:36:02.4546977Z 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:115)
2020-04-06T06:36:02.4547256Z Caused by: java.lang.RuntimeException: host C compiler or linker does not seem to work: java.lang.RuntimeException: returned 1
2020-04-06T06:36:02.4547429Z 
2020-04-06T06:36:02.4548764Z Running command: cc -v -o /project/quarkus-integration-test-quartz-999-SNAPSHOT-runner -z noexecstack -Wl,--gc-sections -Wl,--dynamic-list -Wl,/tmp/SVM-8540717505311960636/exported_symbols.list -Wl,-x -L/tmp/SVM-8540717505311960636 -L/opt/graalvm/jre/lib -L/opt/graalvm/jre/lib/svm/clibraries/linux-amd64 /tmp/SVM-8540717505311960636/quarkus-integration-test-quartz-999-SNAPSHOT-runner.o /opt/graalvm/jre/lib/libzip.a /opt/graalvm/jre/lib/libnet.a /opt/graalvm/jre/lib/libjava.a /opt/graalvm/jre/lib/libnio.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libffi.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/liblibchelper.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libjvm.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libstrictmath.a -lstdc++ -lpthread -ldl -lz -lm -lrt
2020-04-06T06:36:02.4549296Z 
2020-04-06T06:36:02.4549668Z Using built-in specs.
2020-04-06T06:36:02.4549869Z COLLECT_GCC=cc
2020-04-06T06:36:02.4550237Z COLLECT_LTO_WRAPPER=/usr/libexec/gcc/x86_64-redhat-linux/8/lto-wrapper
2020-04-06T06:36:02.4550780Z OFFLOAD_TARGET_NAMES=nvptx-none
2020-04-06T06:36:02.4550987Z OFFLOAD_TARGET_DEFAULT=1
2020-04-06T06:36:02.4551508Z Target: x86_64-redhat-linux
2020-04-06T06:36:02.4552876Z Configured with: ../configure --enable-bootstrap --enable-languages=c,c++,fortran,lto --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=http://bugzilla.redhat.com/bugzilla --enable-shared --enable-threads=posix --enable-checking=release --enable-multilib --with-system-zlib --enable-__cxa_atexit --disable-libunwind-exceptions --enable-gnu-unique-object --enable-linker-build-id --with-gcc-major-version-only --with-linker-hash-style=gnu --enable-plugin --enable-initfini-array --with-isl --disable-libmpx --enable-offload-targets=nvptx-none --without-cuda-driver --enable-gnu-indirect-function --enable-cet --with-tune=generic --with-arch_32=x86-64 --build=x86_64-redhat-linux
2020-04-06T06:36:02.4553260Z Thread model: posix
2020-04-06T06:36:02.4553837Z gcc version 8.3.1 20190507 (Red Hat 8.3.1-4) (GCC) 
2020-04-06T06:36:02.4554555Z COMPILER_PATH=/usr/libexec/gcc/x86_64-redhat-linux/8/:/usr/libexec/gcc/x86_64-redhat-linux/8/:/usr/libexec/gcc/x86_64-redhat-linux/:/usr/lib/gcc/x86_64-redhat-linux/8/:/usr/lib/gcc/x86_64-redhat-linux/
2020-04-06T06:36:02.4556647Z LIBRARY_PATH=/usr/lib/gcc/x86_64-redhat-linux/8/:/usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/:/lib/../lib64/:/usr/lib/../lib64/:/usr/lib/gcc/x86_64-redhat-linux/8/../../../:/lib/:/usr/lib/
2020-04-06T06:36:02.4558405Z COLLECT_GCC_OPTIONS='-v' '-o' '/project/quarkus-integration-test-quartz-999-SNAPSHOT-runner' '-z' 'noexecstack' '-L/tmp/SVM-8540717505311960636' '-L/opt/graalvm/jre/lib' '-L/opt/graalvm/jre/lib/svm/clibraries/linux-amd64' '-mtune=generic' '-march=x86-64'
2020-04-06T06:36:02.4560581Z  /usr/libexec/gcc/x86_64-redhat-linux/8/collect2 -plugin /usr/libexec/gcc/x86_64-redhat-linux/8/liblto_plugin.so -plugin-opt=/usr/libexec/gcc/x86_64-redhat-linux/8/lto-wrapper -plugin-opt=-fresolution=/tmp/ccYCCUrQ.res -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s -plugin-opt=-pass-through=-lc -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s --build-id --no-add-needed --eh-frame-hdr --hash-style=gnu -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 -o /project/quarkus-integration-test-quartz-999-SNAPSHOT-runner -z noexecstack /usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/crt1.o /usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/crti.o /usr/lib/gcc/x86_64-redhat-linux/8/crtbegin.o -L/tmp/SVM-8540717505311960636 -L/opt/graalvm/jre/lib -L/opt/graalvm/jre/lib/svm/clibraries/linux-amd64 -L/usr/lib/gcc/x86_64-redhat-linux/8 -L/usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64 -L/lib/../lib64 -L/usr/lib/../lib64 -L/usr/lib/gcc/x86_64-redhat-linux/8/../../.. --gc-sections --dynamic-list /tmp/SVM-8540717505311960636/exported_symbols.list -x /tmp/SVM-8540717505311960636/quarkus-integration-test-quartz-999-SNAPSHOT-runner.o /opt/graalvm/jre/lib/libzip.a /opt/graalvm/jre/lib/libnet.a /opt/graalvm/jre/lib/libjava.a /opt/graalvm/jre/lib/libnio.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/libsunec.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libffi.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/liblibchelper.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libjvm.a /opt/graalvm/jre/lib/svm/clibraries/linux-amd64/libstrictmath.a -lstdc++ -lpthread -ldl -lz -lm -lrt -lgcc --as-needed -lgcc_s --no-as-needed -lc -lgcc --as-needed -lgcc_s --no-as-needed /usr/lib/gcc/x86_64-redhat-linux/8/crtend.o /usr/lib/gcc/x86_64-redhat-linux/8/../../../../lib64/crtn.o
2020-04-06T06:36:02.4561137Z collect2: error: ld returned 1 exit status
2020-04-06T06:36:02.4561206Z 
2020-04-06T06:36:02.4561295Z 	at com.oracle.svm.hosted.image.NativeBootImageViaCC.write(NativeBootImageViaCC.java:388)
2020-04-06T06:36:02.4561416Z 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:652)
2020-04-06T06:36:02.4561750Z 	at com.oracle.svm.hosted.NativeImageGenerator.lambda$run$0(NativeImageGenerator.java:445)
2020-04-06T06:36:02.4561886Z 	at java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1386)
2020-04-06T06:36:02.4562019Z 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
2020-04-06T06:36:02.4562317Z 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
2020-04-06T06:36:02.4562625Z 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
2020-04-06T06:36:02.4562743Z 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2020-04-06T06:36:02.5877222Z Error: Image build request failed with exit status 1
```

Vault tests seem to be failing for some system reason...

```
2020-04-16T02:50:59.9651564Z [INFO] -------------------------------------------------------
2020-04-16T02:50:59.9655183Z [INFO]  T E S T S
2020-04-16T02:50:59.9657274Z [INFO] -------------------------------------------------------
2020-04-16T02:51:00.7467269Z [INFO] Running io.quarkus.vault.VaultMultiPathConfigITCase
2020-04-16T02:51:03.4230308Z         â„¹ï¸Ž Checking the system...
2020-04-16T02:51:03.4254302Z         âœ” Docker version should be at least 1.6.0
2020-04-16T02:51:03.5743672Z         âœ” Docker environment should have more than 2GB free disk space
2020-04-16T02:51:24.8592775Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 24.079 s <<< FAILURE! - in io.quarkus.vault.VaultMultiPathConfigITCase
2020-04-16T02:51:24.8596104Z [ERROR] io.quarkus.vault.VaultMultiPathConfigITCase  Time elapsed: 24.079 s  <<< ERROR!
2020-04-16T02:51:24.8603239Z java.lang.RuntimeException: Unable to start Quarkus test resource io.quarkus.vault.test.VaultTestLifecycleManager@6b9ce1bf
2020-04-16T02:51:24.8606609Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:38)
2020-04-16T02:51:24.8612775Z 	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:332)
2020-04-16T02:51:24.8659148Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$7(ClassBasedTestDescriptor.java:359)
2020-04-16T02:51:24.8674053Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8674254Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:359)
2020-04-16T02:51:24.8674357Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:189)
2020-04-16T02:51:24.8674671Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
2020-04-16T02:51:24.8680871Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
2020-04-16T02:51:24.8681016Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8681167Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-04-16T02:51:24.8681302Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-04-16T02:51:24.8684386Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-04-16T02:51:24.8684573Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8684673Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-04-16T02:51:24.8684802Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-04-16T02:51:24.8684929Z 	at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-16T02:51:24.8685070Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-04-16T02:51:24.8689583Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
2020-04-16T02:51:24.8689713Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8689987Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-04-16T02:51:24.8690097Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-04-16T02:51:24.8690184Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-04-16T02:51:24.8690301Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8690450Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-04-16T02:51:24.8690584Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-04-16T02:51:24.8693438Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
2020-04-16T02:51:24.8696510Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2020-04-16T02:51:24.8724413Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
2020-04-16T02:51:24.8724652Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
2020-04-16T02:51:24.8724778Z 	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
2020-04-16T02:51:24.8731660Z 	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
2020-04-16T02:51:24.8736620Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
2020-04-16T02:51:24.8736818Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
2020-04-16T02:51:24.8744231Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
2020-04-16T02:51:24.8748137Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:117)
2020-04-16T02:51:24.8748540Z 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
2020-04-16T02:51:24.8748722Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
2020-04-16T02:51:24.8749388Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
2020-04-16T02:51:24.8749740Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2020-04-16T02:51:24.8749889Z 	Suppressed: java.lang.NullPointerException
2020-04-16T02:51:24.8750007Z 		at io.quarkus.test.QuarkusUnitTest.afterAll(QuarkusUnitTest.java:464)
2020-04-16T02:51:24.8753161Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeAfterAllCallbacks$13(ClassBasedTestDescriptor.java:425)
2020-04-16T02:51:24.8753383Z 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8760095Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeAfterAllCallbacks$14(ClassBasedTestDescriptor.java:425)
2020-04-16T02:51:24.8760411Z 		at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-16T02:51:24.8760603Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeAfterAllCallbacks(ClassBasedTestDescriptor.java:425)
2020-04-16T02:51:24.8760767Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.after(ClassBasedTestDescriptor.java:213)
2020-04-16T02:51:24.8765953Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.after(ClassBasedTestDescriptor.java:78)
2020-04-16T02:51:24.8770569Z 		at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:145)
2020-04-16T02:51:24.8770744Z 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-16T02:51:24.8773982Z 		at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:145)
2020-04-16T02:51:24.8777205Z 		... 29 more
2020-04-16T02:51:24.8777384Z Caused by: io.quarkus.vault.VaultException: okhttp3.internal.http2.ConnectionShutdownException
2020-04-16T02:51:24.8777552Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:347)
2020-04-16T02:51:24.8785803Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:332)
2020-04-16T02:51:24.8785969Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.put(OkHttpVaultClient.java:308)
2020-04-16T02:51:24.8786094Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.init(OkHttpVaultClient.java:254)
2020-04-16T02:51:24.8786269Z 	at io.quarkus.vault.test.VaultTestExtension.initVault(VaultTestExtension.java:241)
2020-04-16T02:51:24.8786389Z 	at io.quarkus.vault.test.VaultTestExtension.start(VaultTestExtension.java:230)
2020-04-16T02:51:24.8786493Z 	at io.quarkus.vault.test.VaultTestLifecycleManager.start(VaultTestLifecycleManager.java:32)
2020-04-16T02:51:24.8786646Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:33)
2020-04-16T02:51:24.8786754Z 	... 38 more
2020-04-16T02:51:24.8786886Z Caused by: okhttp3.internal.http2.ConnectionShutdownException
2020-04-16T02:51:24.8787035Z 	at okhttp3.internal.http2.Http2Connection.newStream(Http2Connection.java:255)
2020-04-16T02:51:24.8787180Z 	at okhttp3.internal.http2.Http2Connection.newStream(Http2Connection.java:238)
2020-04-16T02:51:24.8795407Z 	at okhttp3.internal.http2.Http2ExchangeCodec.writeRequestHeaders(Http2ExchangeCodec.java:116)
2020-04-16T02:51:24.8795597Z 	at okhttp3.internal.connection.Exchange.writeRequestHeaders(Exchange.java:72)
2020-04-16T02:51:24.8795761Z 	at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:43)
2020-04-16T02:51:24.8795908Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-16T02:51:24.8796073Z 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:43)
2020-04-16T02:51:24.8796222Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-16T02:51:24.8808380Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-16T02:51:24.8809158Z 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)
2020-04-16T02:51:24.8809422Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-16T02:51:24.8809619Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-16T02:51:24.8809983Z 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
2020-04-16T02:51:24.8814201Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-16T02:51:24.8814978Z 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88)
2020-04-16T02:51:24.8815412Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-16T02:51:24.8830242Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-16T02:51:24.8831611Z 	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229)
2020-04-16T02:51:24.8832203Z 	at okhttp3.RealCall.execute(RealCall.java:81)
2020-04-16T02:51:24.8832964Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:340)
2020-04-16T02:51:24.8837564Z 	... 45 more
```

@vsevel mind taking a look and see if you can reproduce it (I wasn't able to)?

hello @geoand, I had a look, but could not reproduce it either. I executed all vault tests, including the native one. It is failing on `quarkus-integration-test-vault`, on a non native IT, specifically on `VaultAppRoleWrapITCase`, which is the first one to run. In my case it was `VaultClientTokenWrapITCase` to run first, but I think this is irrelevant.

normally we should see
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.quarkus.vault.VaultClientTokenWrapITCase
        ℹ︎ Checking the system...
        ✔ Docker version should be at least 1.6.0
        ✔ Docker environment should have more than 2GB free disk space
2020-04-17 05:30:48,780 DEBUG [io.qua.vau.run.cli.OkHttpClientFactory] (main) create SSLSocketFactory with tls.skip-verify
2020-04-17 05:30:48,852 INFO  [io.qua.vau.tes.VaultTestExtension] (main) start containers on Linux
2020-04-17 05:30:49,291 INFO  [io.qua.vau.tes.VaultTestExtension] (main) starting vault with url=https://localhost:8200 and config file=vault-config-tls.json
2020-04-17 05:30:49,362 INFO  [org.tes.doc.DockerClientProviderStrategy] (main) Loaded org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy from ~/.testcontainers.properties, will try it first
2020-04-17 05:30:50,347 INFO  [org.tes.doc.EnvironmentAndSystemPropertyClientProviderStrategy] (main) Found docker client settings from environment
2020-04-17 05:30:50,347 INFO  [org.tes.doc.DockerClientProviderStrategy] (main) Found Docker environment with Environment variables, system properties and defaults. Resolved dockerHost=unix:///var/run/docker.sock
2020-04-17 05:30:50,537 INFO  [org.tes.DockerClientFactory] (main) Docker host IP address is localhost
2020-04-17 05:30:50,573 INFO  [org.tes.DockerClientFactory] (main) Connected to docker: 
  Server Version: 19.03.4
  API Version: 1.40
  Operating System: Ubuntu 16.04.6 LTS
  Total Memory: 15032 MB
2020-04-17 05:30:50,608 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: quay.io/testcontainers/ryuk:0.2.3, configFile: /home/vvsevel/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/vvsevel/.docker/config.json (No such file or directory)
...
2020-04-17 05:30:53,069 INFO  [org.tes.DockerClientFactory] (main) Ryuk started - will monitor and terminate Testcontainers containers on JVM exit
2020-04-17 05:30:53,536 INFO  [🐳 .6.12]] (main) Creating container for image: postgres:9.6.12
2020-04-17 05:30:53,536 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: postgres:9.6.12, configFile: /home/vvsevel/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/vvsevel/.docker/config.json (No such file or directory)
2020-04-17 05:30:53,550 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: alpine:3.5, configFile: /home/vvsevel/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/vvsevel/.docker/config.json (No such file or directory)
2020-04-17 05:30:55,008 INFO  [org.tes.DockerClientFactory] (tc-okhttp-stream-1666544295) Starting to pull image
2020-04-17 05:30:55,029 INFO  [org.tes.DockerClientFactory] (tc-okhttp-stream-1666544295) Pulling image layers:  0 pending,  0 downloaded,  0 extracted, (0 bytes/0 bytes)
2020-04-17 05:30:56,242 INFO  [org.tes.DockerClientFactory] (tc-okhttp-stream-1666544295) Pulling image layers:  0 pending,  1 downloaded,  0 extracted, (326 KB/1 MB)
2020-04-17 05:30:56,481 INFO  [org.tes.DockerClientFactory] (tc-okhttp-stream-1666544295) Pulling image layers:  0 pending,  1 downloaded,  1 extracted, (1 MB/1 MB)
2020-04-17 05:30:56,494 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: alpine:3.5, configFile: /home/vvsevel/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/vvsevel/.docker/config.json (No such file or directory)
2020-04-17 05:30:56,495 INFO  [org.tes.DockerClientFactory] (tc-okhttp-stream-1666544295) Pull complete. 1 layers, pulled in 1s (downloaded 1 MB at 1 MB/s)
2020-04-17 05:30:58,207 INFO  [🐳 .6.12]] (main) Starting container with ID: 0e0d1d804abf757f21d108ed4613325470739134a09237730bbb2ddbd8635627
2020-04-17 05:30:59,363 INFO  [🐳 .6.12]] (main) Container postgres:9.6.12 is starting: 0e0d1d804abf757f21d108ed4613325470739134a09237730bbb2ddbd8635627
2020-04-17 05:31:08,339 INFO  [🐳 .6.12]] (main) Container postgres:9.6.12 started in PT19.045S
2020-04-17 05:31:08,779 INFO  [io.qua.vau.tes.VaultTestExtension] (main) > psql -U postgres -d mydb -f /tmp/postgres-init.sql
CREATE TABLE

2020-04-17 05:31:08,779 INFO  [🐳 .2.2]] (main) Creating container for image: vault:1.2.2
2020-04-17 05:31:08,779 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: vault:1.2.2, configFile: /home/vvsevel/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/vvsevel/.docker/config.json (No such file or directory)
2020-04-17 05:31:09,947 INFO  [🐳 .2.2]] (main) Starting container with ID: d772bfe437b6cd6d33941c100f6fd8de8cbb31b4a8d5b356d6c5b11f24481a81
2020-04-17 05:31:10,952 INFO  [🐳 .2.2]] (main) Container vault:1.2.2 is starting: d772bfe437b6cd6d33941c100f6fd8de8cbb31b4a8d5b356d6c5b11f24481a81
2020-04-17 05:31:15,296 INFO  [🐳 .2.2]] (main) Container vault:1.2.2 started in PT6.517S
2020-04-17 05:31:16,944 INFO  [io.qua.vau.tes.VaultTestExtension] (main) > vault operator unseal f76b10bb6f5e01024307a2a92b2f780fff485a68de3b33c012ac27ad7a878466
Key             Value
---             -----
Seal Type       shamir
Initialized     true
Sealed          false
Total Shares    1
Threshold       1
Version         1.2.2
Cluster Name    vault-cluster-bd166157
Cluster ID      8ff5af94-0c86-df22-43ad-7138e976d34b
HA Enabled      false
...
2020-04-17 05:31:33,889 DEBUG [io.qua.vau.run.VaultAuthManager] (main) unwrapped client token: s.6eUmgmwpv2K2HqkRkdD7kHfs
2020-04-17 05:31:33,930 INFO  [io.quarkus] (main) Quarkus stopped in 0.019s
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 50.463 s - in io.quarkus.vault.VaultClientTokenWrapITCase
...
```

but instead we see:
```
2020-04-17T03:02:31.3770048Z [INFO] -------------------------------------------------------
2020-04-17T03:02:31.3770460Z [INFO]  T E S T S
2020-04-17T03:02:31.3771344Z [INFO] -------------------------------------------------------
2020-04-17T03:02:32.2451659Z [INFO] Running io.quarkus.vault.VaultAppRoleWrapITCase
2020-04-17T03:02:35.3810679Z         ℹ︎ Checking the system...
2020-04-17T03:02:35.3811591Z         ✔ Docker version should be at least 1.6.0
2020-04-17T03:02:35.5581157Z         ✔ Docker environment should have more than 2GB free disk space
2020-04-17T03:03:39.6586718Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 67.376 s <<< FAILURE! - in io.quarkus.vault.VaultAppRoleWrapITCase
2020-04-17T03:03:39.6587388Z [ERROR] io.quarkus.vault.VaultAppRoleWrapITCase  Time elapsed: 67.376 s  <<< ERROR!
2020-04-17T03:03:39.6628041Z java.lang.RuntimeException: Unable to start Quarkus test resource io.quarkus.vault.test.VaultTestLifecycleManager@1e63ec0b
2020-04-17T03:03:39.6637992Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:38)
2020-04-17T03:03:39.6638172Z 	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:332)
2020-04-17T03:03:39.6638370Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$7(ClassBasedTestDescriptor.java:359)
2020-04-17T03:03:39.6638574Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6638737Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:359)
2020-04-17T03:03:39.6638886Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:189)
2020-04-17T03:03:39.6639032Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
2020-04-17T03:03:39.6639182Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
2020-04-17T03:03:39.6639570Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6639713Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-04-17T03:03:39.6639973Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-04-17T03:03:39.6640094Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-04-17T03:03:39.6640239Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6640380Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-04-17T03:03:39.6640514Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-04-17T03:03:39.6640644Z 	at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-17T03:03:39.6640800Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-04-17T03:03:39.6640964Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
2020-04-17T03:03:39.6641114Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6641255Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-04-17T03:03:39.6641392Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-04-17T03:03:39.6674615Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-04-17T03:03:39.6674882Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6675062Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-04-17T03:03:39.6675206Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-04-17T03:03:39.6675363Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
2020-04-17T03:03:39.6675533Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2020-04-17T03:03:39.6703097Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
2020-04-17T03:03:39.6703749Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
2020-04-17T03:03:39.6704153Z 	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
2020-04-17T03:03:39.6704885Z 	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
2020-04-17T03:03:39.6706464Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
2020-04-17T03:03:39.6706906Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
2020-04-17T03:03:39.6707455Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
2020-04-17T03:03:39.6709198Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:117)
2020-04-17T03:03:39.6711002Z 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
2020-04-17T03:03:39.6714965Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
2020-04-17T03:03:39.6715424Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
2020-04-17T03:03:39.6715793Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2020-04-17T03:03:39.6716401Z 	Suppressed: java.lang.NullPointerException
2020-04-17T03:03:39.6716748Z 		at io.quarkus.test.QuarkusUnitTest.afterAll(QuarkusUnitTest.java:464)
2020-04-17T03:03:39.6718057Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeAfterAllCallbacks$13(ClassBasedTestDescriptor.java:425)
2020-04-17T03:03:39.6719750Z 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6722060Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeAfterAllCallbacks$14(ClassBasedTestDescriptor.java:425)
2020-04-17T03:03:39.6722862Z 		at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-17T03:03:39.6724997Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeAfterAllCallbacks(ClassBasedTestDescriptor.java:425)
2020-04-17T03:03:39.6726781Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.after(ClassBasedTestDescriptor.java:213)
2020-04-17T03:03:39.6736859Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.after(ClassBasedTestDescriptor.java:78)
2020-04-17T03:03:39.6741359Z 		at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:145)
2020-04-17T03:03:39.6741819Z 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-17T03:03:39.6742767Z 		at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:145)
2020-04-17T03:03:39.6743104Z 		... 29 more
2020-04-17T03:03:39.6743437Z Caused by: io.quarkus.vault.VaultException: okhttp3.internal.http2.ConnectionShutdownException
2020-04-17T03:03:39.6743716Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:347)
2020-04-17T03:03:39.6744000Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:332)
2020-04-17T03:03:39.6746492Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.put(OkHttpVaultClient.java:308)
2020-04-17T03:03:39.6747981Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.init(OkHttpVaultClient.java:254)
2020-04-17T03:03:39.6797431Z 	at io.quarkus.vault.test.VaultTestExtension.initVault(VaultTestExtension.java:241)
2020-04-17T03:03:39.6797831Z 	at io.quarkus.vault.test.VaultTestExtension.start(VaultTestExtension.java:230)
2020-04-17T03:03:39.6798191Z 	at io.quarkus.vault.test.VaultTestLifecycleManager.start(VaultTestLifecycleManager.java:32)
2020-04-17T03:03:39.6800112Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:33)
2020-04-17T03:03:39.6800421Z 	... 38 more
2020-04-17T03:03:39.6801846Z Caused by: okhttp3.internal.http2.ConnectionShutdownException
2020-04-17T03:03:39.6802142Z 	at okhttp3.internal.http2.Http2Connection.newStream(Http2Connection.java:255)
2020-04-17T03:03:39.6803402Z 	at okhttp3.internal.http2.Http2Connection.newStream(Http2Connection.java:238)
2020-04-17T03:03:39.6805203Z 	at okhttp3.internal.http2.Http2ExchangeCodec.writeRequestHeaders(Http2ExchangeCodec.java:116)
2020-04-17T03:03:39.6806350Z 	at okhttp3.internal.connection.Exchange.writeRequestHeaders(Exchange.java:72)
2020-04-17T03:03:39.6807497Z 	at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:43)
2020-04-17T03:03:39.6813227Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-17T03:03:39.6813586Z 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:43)
2020-04-17T03:03:39.6813872Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-17T03:03:39.6814157Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-17T03:03:39.6814427Z 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)
2020-04-17T03:03:39.6814730Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-17T03:03:39.6815266Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-17T03:03:39.6821029Z 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
2020-04-17T03:03:39.6821400Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-17T03:03:39.6821919Z 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88)
2020-04-17T03:03:39.6822203Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-17T03:03:39.6829545Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-17T03:03:39.6834887Z 	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229)
2020-04-17T03:03:39.6838463Z 	at okhttp3.RealCall.execute(RealCall.java:81)
2020-04-17T03:03:39.6867938Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:340)
2020-04-17T03:03:39.6869950Z 	... 45 more
```

the `VaultTestExtension` first starts the vault and postgresql containers, then execute some commands on vault. and the first command is a vault `init`, which is failing here.

notice it took 67 secs for the test to fail. so I think we start the containers probably successfully (it would nice to have the extra INFO logs), and call the `init` endpoint, but for some reason get stuck on it. eventually the connection times out (not sure if this is a read timeout, or a connection timeout) and we get the `ConnectionShutdownException`.

later we can see another test failing with:
```
Caused by: com.github.dockerjava.api.exception.InternalServerErrorException: {"message":"driver failed programming external connectivity on endpoint funny_lamport (0918dd8d16c72cd710bc6950867cafa0c7547172290fc6d48c230c3155b7f6fe): Bind for 0.0.0.0:6543 failed: port is already allocated"}
```

this is interesting because `6543` is the port of the postgresql container. this means we were able to get it to run successfully.

so there is an issue with the vault container. it either died, or does not accept connections, or does not return a response on the `init` call. probably one of the last 2, because I do not think we would wait 67 secs trying to connect on a non existent process.

if we want to progress on this, it would be nice to have 
 - the extra INFO logs in the build, 
 - the state of the vault container and its logs

do we know when it started failing? after what commit? were there environmental changes?

@geoand from what I see, it was still good 3 days ago: 

https://github.com/quarkusio/quarkus/runs/584397198?check_suite_focus=true
Merge pull request #8409 from stuartwdouglas/7809 master stuartwdouglas  084571e
[raw log](https://pipelines.actions.githubusercontent.com/5Na9LjmLgXLiLuzJ0Z1qHRbIDVUXUHNrKe6jPcqGdw5t1S8TiW/_apis/pipelines/1/runs/17190/signedlogcontent/3?urlExpires=2020-04-17T07%3A07%3A18.8258493Z&urlSigningMethod=HMACV1&urlSignature=rhQqFHjJWKa0H27mjClrZ2GdPh34LSGRW9FtMjMQeCI%3D)

```
...
2020-04-14T03:04:13.1928532Z 2020-04-14 03:04:01,240 INFO  [ðŸ³ .2.2]] (main) Container vault:1.2.2 is starting: ac2afa72a9ba768d25d441736c694f6fb6e557cdd3f5a13c0aab5d063f038c69
2020-04-14T03:04:13.1929912Z 2020-04-14 03:04:02,727 INFO  [ðŸ³ .2.2]] (main) Container vault:1.2.2 started in PT7.044S
2020-04-14T03:04:13.1932217Z 2020-04-14 03:04:03,397 INFO  [io.qua.vau.tes.VaultTestExtension] (main) > vault operator unseal 0ebfc378aafc7515a928d541d73fc7ecc96120cb11aa58d87797d2dbc1402854
2020-04-14T03:04:13.1933302Z Key             Value
2020-04-14T03:04:13.1934444Z ---             -----
2020-04-14T03:04:13.1935427Z Seal Type       shamir
2020-04-14T03:04:13.1936356Z Initialized     true
2020-04-14T03:04:13.1937261Z Sealed          false
2020-04-14T03:04:13.1938025Z Total Shares    1
2020-04-14T03:04:13.1938535Z Threshold       1
2020-04-14T03:04:13.1939030Z Version         1.2.2
2020-04-14T03:04:13.1940600Z Cluster Name    vault-cluster-4adf4c19
2020-04-14T03:04:13.1942126Z Cluster ID      ba9d60bb-6aa5-537f-d990-5fe3813a050e
2020-04-14T03:04:13.1943398Z HA Enabled      false
...
```

and it was not 2 days ago:

https://github.com/quarkusio/quarkus/runs/587628594?check_suite_focus=true
Merge pull request #8116 from machi1990/refactor/review-service-locat… master gsmet  7894a22
[raw log](https://pipelines.actions.githubusercontent.com/5Na9LjmLgXLiLuzJ0Z1qHRbIDVUXUHNrKe6jPcqGdw5t1S8TiW/_apis/pipelines/1/runs/17522/signedlogcontent/3?urlExpires=2020-04-17T07%3A07%3A22.6819987Z&urlSigningMethod=HMACV1&urlSignature=QgzKRV2adwJo0QGW2uI2gKrKpmsvylRq%2Br%2BHQGAvaSA%3D)

```
2020-04-15T02:56:28.1873448Z [INFO] Running io.quarkus.vault.VaultMultiPathConfigITCase
2020-04-15T02:56:30.9731501Z         â„¹ï¸Ž Checking the system...
2020-04-15T02:56:30.9736970Z         âœ” Docker version should be at least 1.6.0
2020-04-15T02:56:31.0658458Z         âœ” Docker environment should have more than 2GB free disk space
2020-04-15T02:57:24.7597448Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 56.542 s <<< FAILURE! - in io.quarkus.vault.VaultMultiPathConfigITCase
2020-04-15T02:57:24.7602036Z [ERROR] io.quarkus.vault.VaultMultiPathConfigITCase  Time elapsed: 56.542 s  <<< ERROR!
2020-04-15T02:57:24.7604177Z java.lang.RuntimeException: Unable to start Quarkus test resource io.quarkus.vault.test.VaultTestLifecycleManager@160ac7fb
2020-04-15T02:57:24.7622147Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:38)
2020-04-15T02:57:24.7625494Z 	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:332)
...
```

in between those commits we had:
```
7894a228a Merge pull request #8116 from machi1990/refactor/review-service-locators-scanning-for-liquibase-extension
328e5d084 Merge pull request #7898 from stuartwdouglas/access-logging
8625aad8f Merge pull request #8544 from geoand/cron-build-fix
de943e520 Merge pull request #8565 from michalszynkiewicz/rm-2.0
79aeb5060 Merge pull request #8572 from geoand/jbang-report
8ca1d0d38 Merge pull request #7987 from Ladicek/opentracing-tests-and-fixes
074168dea refactor(liquibase): record service classes implementation instead of generating a native resource file for each service interface
0769017db Merge pull request #8420 from gsmet/rest-client-provider-fix
02df24696 Replace kscript for the native cron report with jbang
6b40da8cc Merge pull request #8433 from 38leinaD/gradle-quarkusdev-args-for-command-mode
06dc69582 Merge pull request #8536 from mkouba/vertx-web-route-perf
356f1c7a7 SmallRye Reactive Messaging 2.0 - rework
62c4bf38c Revert how readers and writers are registered for REST clients
6d293be9a Reactive routes - improve @Route method invocation performance
8334a906c SmallRye Reactive Messaging 2.0 - blocking support + tests
fcf329b01 Merge pull request #8520 from jharting/kafka-streams-guide-fix-compiler-errors
09e78592b Switch to JBoss Logging in Kafka Streams guide
f6aac5762 Merge pull request #8557 from cescoffier/features/add-link-to-ubi
b1ccb70fe Merge pull request #8552 from gemmellr/proton-j-update
88d3e6b39 Add an admonition about UBI
31430184e Merge pull request #8278 from sberyozkin/client_secret_jwt
084571ec2 Merge pull request #8409 from stuartwdouglas/7809
```

so it is either in those changes (but nothing looks really suspicious?), or it is environment related between Tue, 14 Apr 2020 05:00:58 GMT and Wed, 15 Apr 2020 02:56:22 GMT.

Still the same issue with the Vault tests:

```
2020-04-21T02:59:02.8309454Z [INFO] Running io.quarkus.vault.VaultAppRoleWrapITCase
2020-04-21T02:59:06.5601173Z         â„¹ï¸Ž Checking the system...
2020-04-21T02:59:06.5602030Z         âœ” Docker version should be at least 1.6.0
2020-04-21T02:59:06.6841730Z         âœ” Docker environment should have more than 2GB free disk space
2020-04-21T02:59:59.6751264Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 56.825 s <<< FAILURE! - in io.quarkus.vault.VaultAppRoleWrapITCase
2020-04-21T02:59:59.6757641Z [ERROR] io.quarkus.vault.VaultAppRoleWrapITCase  Time elapsed: 56.824 s  <<< ERROR!
2020-04-21T02:59:59.6757992Z java.lang.RuntimeException: Unable to start Quarkus test resource io.quarkus.vault.test.VaultTestLifecycleManager@36546a22
2020-04-21T02:59:59.6765428Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:38)
2020-04-21T02:59:59.6767592Z 	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:332)
2020-04-21T02:59:59.6771019Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$7(ClassBasedTestDescriptor.java:359)
2020-04-21T02:59:59.6774094Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.6777254Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:359)
2020-04-21T02:59:59.6780327Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:189)
2020-04-21T02:59:59.6782977Z 	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
2020-04-21T02:59:59.6785702Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
2020-04-21T02:59:59.6788038Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.6790422Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-04-21T02:59:59.7025414Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-04-21T02:59:59.7056997Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-04-21T02:59:59.7061673Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.7062524Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-04-21T02:59:59.7064636Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-04-21T02:59:59.7132208Z 	at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-21T02:59:59.7132733Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
2020-04-21T02:59:59.7133127Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
2020-04-21T02:59:59.7133438Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.7133747Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
2020-04-21T02:59:59.7134068Z 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
2020-04-21T02:59:59.7134367Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
2020-04-21T02:59:59.7138833Z 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.7139016Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
2020-04-21T02:59:59.7139151Z 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
2020-04-21T02:59:59.7140100Z 	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
2020-04-21T02:59:59.7140264Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
2020-04-21T02:59:59.7140398Z 	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
2020-04-21T02:59:59.7140533Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
2020-04-21T02:59:59.7140677Z 	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
2020-04-21T02:59:59.7140813Z 	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
2020-04-21T02:59:59.7140949Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
2020-04-21T02:59:59.7141080Z 	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
2020-04-21T02:59:59.7141221Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
2020-04-21T02:59:59.7141352Z 	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:117)
2020-04-21T02:59:59.7141834Z 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
2020-04-21T02:59:59.7141984Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
2020-04-21T02:59:59.7142189Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
2020-04-21T02:59:59.7142443Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2020-04-21T02:59:59.7142562Z 	Suppressed: java.lang.NullPointerException
2020-04-21T02:59:59.7142685Z 		at io.quarkus.test.QuarkusUnitTest.afterAll(QuarkusUnitTest.java:465)
2020-04-21T02:59:59.7142814Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeAfterAllCallbacks$13(ClassBasedTestDescriptor.java:425)
2020-04-21T02:59:59.7142958Z 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.7143105Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeAfterAllCallbacks$14(ClassBasedTestDescriptor.java:425)
2020-04-21T02:59:59.7143719Z 		at java.util.ArrayList.forEach(ArrayList.java:1257)
2020-04-21T02:59:59.7143863Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeAfterAllCallbacks(ClassBasedTestDescriptor.java:425)
2020-04-21T02:59:59.7144009Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.after(ClassBasedTestDescriptor.java:213)
2020-04-21T02:59:59.7144146Z 		at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.after(ClassBasedTestDescriptor.java:78)
2020-04-21T02:59:59.7144287Z 		at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:145)
2020-04-21T02:59:59.7144411Z 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2020-04-21T02:59:59.7144546Z 		at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:145)
2020-04-21T02:59:59.7144668Z 		... 29 more
2020-04-21T02:59:59.7144818Z Caused by: io.quarkus.vault.VaultException: okhttp3.internal.http2.ConnectionShutdownException
2020-04-21T02:59:59.7144960Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:347)
2020-04-21T02:59:59.7145095Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:332)
2020-04-21T02:59:59.7145222Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.put(OkHttpVaultClient.java:308)
2020-04-21T02:59:59.7145338Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.init(OkHttpVaultClient.java:254)
2020-04-21T02:59:59.7145471Z 	at io.quarkus.vault.test.VaultTestExtension.initVault(VaultTestExtension.java:241)
2020-04-21T02:59:59.7145601Z 	at io.quarkus.vault.test.VaultTestExtension.start(VaultTestExtension.java:230)
2020-04-21T02:59:59.7145735Z 	at io.quarkus.vault.test.VaultTestLifecycleManager.start(VaultTestLifecycleManager.java:32)
2020-04-21T02:59:59.7145865Z 	at io.quarkus.test.common.TestResourceManager.start(TestResourceManager.java:33)
2020-04-21T02:59:59.7145981Z 	... 38 more
2020-04-21T02:59:59.7146099Z Caused by: okhttp3.internal.http2.ConnectionShutdownException
2020-04-21T02:59:59.7172168Z 	at okhttp3.internal.http2.Http2Connection.newStream(Http2Connection.java:255)
2020-04-21T02:59:59.7172371Z 	at okhttp3.internal.http2.Http2Connection.newStream(Http2Connection.java:238)
2020-04-21T02:59:59.7172522Z 	at okhttp3.internal.http2.Http2ExchangeCodec.writeRequestHeaders(Http2ExchangeCodec.java:116)
2020-04-21T02:59:59.7172673Z 	at okhttp3.internal.connection.Exchange.writeRequestHeaders(Exchange.java:72)
2020-04-21T02:59:59.7172824Z 	at okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:43)
2020-04-21T02:59:59.7173089Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-21T02:59:59.7173215Z 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:43)
2020-04-21T02:59:59.7173347Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-21T02:59:59.7173608Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-21T02:59:59.7173750Z 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)
2020-04-21T02:59:59.7173937Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-21T02:59:59.7174069Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-21T02:59:59.7174204Z 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
2020-04-21T02:59:59.7174323Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-21T02:59:59.7174469Z 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88)
2020-04-21T02:59:59.7194166Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
2020-04-21T02:59:59.7195067Z 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
2020-04-21T02:59:59.7197419Z 	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229)
2020-04-21T02:59:59.7197677Z 	at okhttp3.RealCall.execute(RealCall.java:81)
2020-04-21T02:59:59.7197959Z 	at io.quarkus.vault.runtime.client.OkHttpVaultClient.exec(OkHttpVaultClient.java:340)
2020-04-21T02:59:59.7198068Z 	... 45 more
```

@evanchooly @FroMage we are seeing `quarkus-integration-test-hibernate-orm-panache-kotlin` fail when building a native image with Java 8.

Here is the output:

```
2020-05-14T03:45:30.9172945Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on GraalVM Version 19.3.1 CE
2020-05-14T03:45:31.7715881Z [INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] docker run -v /home/runner/work/quarkus/quarkus/integration-tests/hibernate-orm-panache-kotlin/target/quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-native-image-source-jar:/project:z --env LANG=C --user 1001:116 --rm quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java8 -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-DCoordinatorEnvironmentBean.transactionStatusManagerEnable=false -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=1 -J-Dcom.sun.xml.bind.v2.bytecode.ClassTailor.noOptimize=true -J-Duser.language=en -J-Dfile.encoding=UTF-8 --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime -H:+JNI -jar quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner.jar -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:-AddAllCharsets -H:-IncludeAllTimeZones -H:EnableURLProtocols=http,https --enable-all-security-services --no-server -H:-UseServiceLoaderFeature -H:+StackTrace quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner
2020-05-14T03:45:54.7897643Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]    classlist:  21,579.70 ms
2020-05-14T03:45:55.2989530Z 03:45:55,286 INFO  [com.sun.xml.bin.v2.run.ref.opt.AccessorInjector] The optimized code generation is disabled
2020-05-14T03:45:56.7445930Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]        (cap):   1,346.44 ms
2020-05-14T03:45:58.2047480Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]        setup:   3,406.18 ms
2020-05-14T03:45:59.6889763Z 03:45:59,687 INFO  [org.hib.Version] HHH000412: Hibernate ORM core version 5.4.15.Final
2020-05-14T03:45:59.6974072Z 03:45:59,696 INFO  [org.hib.ann.com.Version] HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-05-14T03:45:59.7381132Z 03:45:59,737 INFO  [org.hib.dia.Dialect] HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
2020-05-14T03:46:45.1583893Z 03:46:45,157 INFO  [org.jbo.threads] JBoss Threads version 3.1.1.Final
2020-05-14T03:48:57.0301052Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]   (typeflow): 114,941.80 ms
2020-05-14T03:48:57.0877735Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]    (objects):  55,769.34 ms
2020-05-14T03:48:57.0916161Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]   (features):   1,900.54 ms
2020-05-14T03:48:58.5935451Z [quarkus-integration-test-hibernate-orm-panache-kotlin-999-SNAPSHOT-runner:20]     analysis: 180,046.39 ms
2020-05-14T03:48:58.6420312Z Error: Unsupported features in 12 methods
2020-05-14T03:48:58.6423690Z Detailed message:
2020-05-14T03:48:58.6430494Z Error: Class initialization of org.h2.value.ValueGeometry failed. Use the option --initialize-at-run-time=org.h2.value.ValueGeometry to explicitly request delayed initialization of this class.
2020-05-14T03:48:58.6434404Z Original exception that caused the problem: java.lang.NoClassDefFoundError: org/locationtech/jts/geom/CoordinateSequenceFilter
2020-05-14T03:48:58.6441140Z 	at sun.misc.Unsafe.ensureClassInitialized(Native Method)
2020-05-14T03:48:58.6446399Z 	at com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.ensureClassInitialized(ConfigurableClassInitialization.java:167)
2020-05-14T03:48:58.6452890Z 	at com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.computeInitKindAndMaybeInitializeClass(ConfigurableClassInitialization.java:560)
2020-05-14T03:48:58.6460888Z 	at com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.computeInitKindAndMaybeInitializeClass(ConfigurableClassInitialization.java:130)
2020-05-14T03:48:58.6473484Z 	at com.oracle.svm.hosted.classinitialization.ConfigurableClassInitialization.maybeInitializeHosted(ConfigurableClassInitialization.java:158)
2020-05-14T03:48:58.6802161Z 	at com.oracle.svm.hosted.SVMHost.registerType(SVMHost.java:198)
2020-05-14T03:48:58.6806738Z 	at com.oracle.graal.pointsto.meta.AnalysisUniverse.createType(AnalysisUniverse.java:264)
2020-05-14T03:48:58.6814720Z 	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:205)
2020-05-14T03:48:58.6825137Z 	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:182)
2020-05-14T03:48:58.6831903Z 	at com.oracle.graal.pointsto.meta.AnalysisMethod.getDeclaringClass(AnalysisMethod.java:344)
2020-05-14T03:48:58.6839621Z 	at com.oracle.graal.pointsto.meta.AnalysisMethod.getSignature(AnalysisMethod.java:313)
2020-05-14T03:48:58.6843944Z 	at com.oracle.graal.pointsto.flow.MethodFlowsGraph.<init>(MethodFlowsGraph.java:98)
2020-05-14T03:48:58.6844250Z 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.<init>(MethodTypeFlow.java:75)
2020-05-14T03:48:58.6850457Z 	at com.oracle.graal.pointsto.meta.AnalysisMethod.<init>(AnalysisMethod.java:134)
2020-05-14T03:48:58.6852885Z 	at com.oracle.graal.pointsto.meta.AnalysisUniverse.createMethod(AnalysisUniverse.java:412)
2020-05-14T03:48:58.6858167Z 	at com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:400)
2020-05-14T03:48:58.6861505Z 	at com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.lookupMethod(WrappedConstantPool.java:116)
2020-05-14T03:48:58.6867067Z 	at org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4168)
2020-05-14T03:48:58.6868789Z 	at org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1548)
2020-05-14T03:48:58.6875252Z 	at org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5170)
2020-05-14T03:48:58.6879467Z 	at org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3286)
2020-05-14T03:48:58.6879685Z 	at org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3093)
2020-05-14T03:48:58.6879902Z 	at org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:977)
2020-05-14T03:48:58.6880109Z 	at org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:871)
2020-05-14T03:48:58.6887062Z 	at org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
2020-05-14T03:48:58.6889567Z 	at org.graalvm.compiler.phases.Phase.run(Phase.java:49)
2020-05-14T03:48:58.6891669Z 	at org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:197)
2020-05-14T03:48:58.6891891Z 	at org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
2020-05-14T03:48:58.6892241Z 	at org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
2020-05-14T03:48:58.7272052Z 	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:221)
2020-05-14T03:48:58.7278433Z 	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:340)
2020-05-14T03:48:58.7282071Z 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.doParse(MethodTypeFlow.java:310)
2020-05-14T03:48:58.7288042Z 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureParsed(MethodTypeFlow.java:300)
2020-05-14T03:48:58.7288345Z 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.addContext(MethodTypeFlow.java:107)
2020-05-14T03:48:58.7290328Z 	at com.oracle.graal.pointsto.flow.StaticInvokeTypeFlow.update(InvokeTypeFlow.java:346)
2020-05-14T03:48:58.7290655Z 	at com.oracle.graal.pointsto.BigBang$2.run(BigBang.java:511)
2020-05-14T03:48:58.7293982Z 	at com.oracle.graal.pointsto.util.CompletionExecutor.lambda$execute$0(CompletionExecutor.java:171)
2020-05-14T03:48:58.7294216Z 	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
2020-05-14T03:48:58.7297426Z 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
2020-05-14T03:48:58.7302090Z 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
2020-05-14T03:48:58.7302338Z 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
2020-05-14T03:48:58.7310215Z 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2020-05-14T03:48:58.7310651Z Caused by: java.lang.ClassNotFoundException: org.locationtech.jts.geom.CoordinateSequenceFilter
2020-05-14T03:48:58.7310908Z 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
2020-05-14T03:48:58.7314538Z 	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
2020-05-14T03:48:58.7315547Z 	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
2020-05-14T03:48:58.7318543Z 	... 42 more
```

--- a/integration-tests/hibernate-orm-panache-kotlin/pom.xml
+++ b/integration-tests/hibernate-orm-panache-kotlin/pom.xml
@@ -36,11 +36,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-jdbc-postgresql</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.postgresql</groupId>
-            <artifactId>postgresql</artifactId>
+            <artifactId>quarkus-jdbc-h2</artifactId>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>

